.circleci/config.yml

```diff
@@ -1,25 +1,30 @@
 version: 2.1
-orbs:
-  node: circleci/node@1.1.6
 jobs:
-  build-and-test:
-    executor:
-      name: node/default
-      tag: '12-browsers'
+  build:
+    docker:
+      - image: circleci/node:12-browsers
     steps:
       - checkout
-      - node/with-cache:
-          steps:
-            - run: npm install
-            - run: npm run lint
-            - run: npm run build
-            - run: npm run test-ci
-            - run:
-                name: Upload coverage
-                command: ./node_modules/.bin/codecov
+      - restore_cache:
+          key: dependencies-{{ checksum "package-lock.json" }}
+      - run: npm ci
+      - save_cache:
+          key: dependencies-{{ checksum "package-lock.json" }}
+          paths:
+            - ~/.npm
+            - ~/.cache
```

Comment:
had to move to this approach for the cypress binary to cache correctly

Reply:
I removed the "build-and-test" check requirement from the GitHub project 👍